### DISCUSSION TITLE
git lfs track for files with no extension

### DISCUSSION THREAD

**Discussion by RonRahaman** [git-lfs/git-lfs#5070] [Q&A]
I'm interested in using LFS to track files with *no* extension.  Basically, they're expected to have no `.` in their filename.  Is there a pattern for `git lfs track` that could do this?  

I tried looking at the docs about patterns [here](https://git-scm.com/docs/gitattributes), but it wasn't clear to me.

Thanks!

**Comment by bk2204** [comment]
Hey,

You can't do this with `git lfs track`, but you can probably do this with the following `.gitattributes` patterns (completely untested):

```
* filter=lfs diff=lfs merge=lfs -text
*.* !filter !diff !merge text=auto
```
